### PR TITLE
Tag RPC Service

### PR DESCRIPTION
**Pull request by JesusMtnez**
Manage Tags using RPC service / client.

### CODE REVIEW

File: modules/examples/todolist/client/src/main/scala/ClientProgram.scala [hunk 1]
```diff
@@ -17,18 +17,27 @@
...
-  def clientProgram[M[_]: Monad](implicit client: PingPongClient[M]): M[Unit] =
+  def pongProgram[M[_]: Monad](implicit client: PingPongClient[M]): M[Unit] =
     client.ping()
+
+  def tagProgram[M[_]: Monad](implicit tagClient: TagClient[M]): M[Unit] =
+    for {
+      _         <- tagClient.reset()
+      optionTag <- tagClient.insert(TagRequest("tag"))
+    } yield {
+      //println(s">>> $optionTag")
```
Comment (AdrianRaFo) [inline modules/examples/todolist/client/src/main/scala/ClientProgram.scala]: :fire:

Comment (JesusMtnez) [reply modules/examples/todolist/client/src/main/scala/ClientProgram.scala]: My bad! :disappointed:

File: project/ProjectPlugin.scala [hunk 2]
```diff
@@ -23,7 +23,7 @@ object ProjectPlugin extends AutoPlugin {
       val avrohugger: String         = "1.0.0-RC4"
       val catsEffect: String         = "0.10.1"
       val circe: String              = "0.9.3"
-      val frees: String              = "0.8.0"
+      val frees: String              = "0.8.1-SNAPSHOT"
```
Comment (juanpedromoreno) [inline project/ProjectPlugin.scala]: This should be 0.8.0

Comment (JesusMtnez) [reply project/ProjectPlugin.scala]: Totally agree! It's better this way

File: project/ProjectPlugin.scala [hunk 3]
```diff
@@ -164,6 +164,7 @@ object ProjectPlugin extends AutoPlugin {
 
     lazy val exampleTodolistCommonSettings: Seq[Def.Setting[_]] = Seq(
       libraryDependencies ++= Seq(
+        "io.frees" %% "frees-todolist-lib" % V.frees,
```
[the remark this answers is not in the document]
Comment (JesusMtnez) [reply project/ProjectPlugin.scala]: Done

File: modules/examples/todolist/client/src/main/scala/ClientApp.scala [hunk 4]
```diff
@@ -31,7 +31,8 @@ object ClientApp {
 
     logger.info(s"${Thread.currentThread().getName} Starting client...")
 
-    clientProgram[IO].unsafeRunSync()
+    pongProgram[IO].unsafeRunSync()
+    tagProgram[IO].unsafeRunSync()
```
Comment (juanpedromoreno) [inline modules/examples/todolist/client/src/main/scala/ClientApp.scala]: You could run both programs with just one effect, for example:

```scala
import cats.syntax.apply._
(pongProgram[IO] *> tagProgram[IO]).unsafeRunSync()
```

Comment (JesusMtnez) [reply modules/examples/todolist/client/src/main/scala/ClientApp.scala]: Thanks!

**Pull request merged**